packages/node_modules/@ciscospark/sparkd/README.md

```diff
@@ -1,97 +1,8 @@
+## WARNING
 
-> HTTP frontend to the Cisco Webex JS SDK
-
-Why would we put an http server in front of our SDK? Encryption is hard and this lets all of our client test suites (potentially written in languages for which we do not have sdks) do encrypted things without a major time expenditure
-
-- [Install](#install)
-- [Usage](#usage)
-- [Contribute](#contribute)
-- [Maintainers](#maintainers)
-- [License](#license)
-
-## Install
-
-```bash
-  npm install -g @ciscospark/sparkd
-```
-
-## Usage
-
-Start the daemon
-
-```bash
-sparkd
-```
-
-Create a session (make sure to copy your set cookie header)
-
-```bash
-curl -X POST \
-  -H "Content-Type: application/json" \
-  -d '{"clientId":"<your client id>","clientSecret":"<your client secret>","redirectUri":"<your  redirect_uri>","scope":"<your scopes>"}' \
-  http://localhost:3000/api/v1/session
-```
-
-(optional) Create a conversation
-
-> SDK: spark.internal.conversation.create({comment: 'first comment', displayName: 'title', participants: ['<userId1>', '<userId2>', '<userId3>']})
-
-```
-curl -X POST \
-  -H "Content-Type: application/json" \
-  -H "Cookie: <connect.sid cookie from step one>" \
-  -d '[{"comment":"first message","displayName":"title","participants":["userId1","userId2","userId3"]}]'
-  -v \
-  http://localhost:3000/api/v1/session/invoke/internal/conversation/create
-```
-
-(optional) Post a message
-
-> SDK: `spark.inernal.conversation.post({url: '<conversation url>', {displayName: 'second comment'}})`
-
-```
-curl -X POST \
-  -H "Content-Type: application/json" \
-  -H "Cookie: <connect.sid cookie from step one>" \
-  -d [{"url":"<conversation url>"},{"displayName":"second comment"}]
-  -v \
-  http://localhost:3000/api/v1/session/invoke/internal/conversation/post
-```
-
-(optional) Fetch a conversation
-
-> SDK: `spark.internal.conversation.get({url: '<conversation url>'})`
-> SDK: `spark.internal.conversation.get({url: '<conversation url>'})`
-
-```
-curl -X POST \
-  -H "Content-Type: application/json" \
-  -H "Cookie: <connect.sid cookie from step one>" \
-  -d [{"url":"<conversation url>"}]
-  -v \
-  http://localhost:3000/api/v1/session/invoke/internal/conversation/get
-```
-
-Clean up your session (If you don't do this, you'll have a bunch of long-running web socket connections)
-
-```
-curl -X DELETE \
-  -H "Content-Type: application/json" \
-  -H "Cookie: <connect.sid cookie from step one>" \
-  -v \
-  http://localhost:3000/api/v1/session
-```
-
-## Maintainers
-
-This package is maintained by [Cisco Webex for Developers](https://developer.webex.com/).
-
-## Contribute
-
-Pull requests welcome. Please see [CONTRIBUTING.md](https://github.com/webex/spark-js-sdk/blob/master/CONTRIBUTING.md) for more details.
+We renamed this module to [@webex/sparkd](https://www.npmjs.com/package/@webex/sparkd). Please install it instead.
```

Comment:
Should we have the link to the github repo as well?

Reply:
Once you get over to the npm page there is a link to it. The truly last step in deprecation is removing the old code. I don't think this will unpublish the existing modules on npm, but I do want to test that out. 

Essentially, I don't want to disrupt the old modules for at least some length of time. I think they would always exist on npm but push you over to the new modules and repo.